lib/application/presentation/detail_screen.dart

```diff
@@ -1,25 +1,294 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+import '../../constants/my_colors.dart';
+import '../../constants/schedule_item_categories.dart';
+import '../../domain/models/speaker.dart';
+import '../logic/speaker_cubit.dart';
 
 class DetailScreen extends StatefulWidget {
   const DetailScreen({super.key, required this.scheduleItem});
+
   final ScheduleItem scheduleItem;
 
   @override
   State<DetailScreen> createState() => _DetailScreenState();
 }
 
 class _DetailScreenState extends State<DetailScreen> {
+  @override
+  void initState() {
+    super.initState();
+    context
+        .read<SpeakerCubit>()
+        .readSpeakerItems([widget.scheduleItem.speakerId]);
+  }
+
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      appBar: AppBar(
-        title: Text(
-          'Schedule Item Detail',
-          style: Theme.of(context).textTheme.titleLarge,
-        ),
-      ),
-      body: Text(widget.scheduleItem.description, style: Theme.of(context).textTheme.titleMedium),
-    );
+    return Scaffold(body: BlocBuilder<SpeakerCubit, SpeakerState>(
+      builder: (context, state) {
+        return Column(
+          children: [
+            Stack(
+              children: [
+                Image.asset('images/background_details.png',
+                    alignment: Alignment.centerLeft,
+                    fit: BoxFit.cover,
+                    height: 240,
+                    width: double.infinity),
+                Column(children: [
+                  Padding(
+                      padding: const EdgeInsets.only(
+                          top: 50, left: 40, right: 40, bottom: 10),
+                      child: Row(
+                        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        children: [
+                          Wrap(
+                            spacing: 5,
+                            children: [
+                              GestureDetector(
+                                onTap: () {
+                                  Navigator.pop(context);
+                                },
+                                child: const Icon(Icons.arrow_back,
+                                    size: 30, color: MyColors.color9B9A9B),
+                              ),
+                            ],
+                          ),
+                          Wrap(
+                            spacing: 5,
+                            children: [
+                              GestureDetector(
+                                onTap: () {},
+                                child: const Icon(Icons.add,
+                                    size: 30, color: MyColors.color9B9A9B),
+                              ),
+                            ],
+                          )
+                        ],
+                      )),
+                  const Divider(
```

Comment:
Vidim da si više puta na ovom ekranu ovako Divider s istim argumentima koristio. Onda ga izdvoji u Widget, možeš ga nazvati nešto tipa MyHorizontalDivider. Tip, možda ti bude lakše implementirati ga uz pomoć Container widgeta nego koristiti built in Divider